libpromises/eval_context.c

```diff
@@ -1443,6 +1450,8 @@ static bool EvalContextClassPut(EvalContext *ctx, const char *ns, const char *na
         return false;
     }
 
+    StringSetAdd(ctx->all_classes, SafeStringDuplicate(name));
```

Comment:
I believe this pull request does not offer any functionality to Core, but this one line adds a lot of memory of overhead, because of an Enterprise-only feature. Can we make it optional?